sources/packages/forms/src/form-definitions/studentrequestchange.json

```diff
@@ -455,6 +455,10 @@
               {
                 "value": "studentdisabilityappeal",
                 "label": "Disability Information"
+              },
```

Comment:
IMO we should ensure the alphabetical order in this list.
I noticed that the "Disability Information" was already added not following the order, but I believe that we keep some sort of order.

Reply:
Order updated. 
![image](https://github.com/bcgov/SIMS/assets/54600590/a26df868-0709-4691-a40a-bbda1d5ac1f4)